csharp/ql/test/resources/stubs/_frameworks/Microsoft.NETCore.App/System.Runtime.cs

```diff
@@ -5994,7 +5994,7 @@ public class Stopwatch
 
         namespace CodeAnalysis
         {
-            // Generated from `System.Diagnostics.CodeAnalysis.AllowNullAttribute` in `System.Diagnostics.DiagnosticSource, Version=5.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51; System.Formats.Asn1, Version=5.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51; System.Runtime, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a; System.Threading.Tasks.Dataflow, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a`
+            // Generated from `System.Diagnostics.CodeAnalysis.AllowNullAttribute` in `Newtonsoft.Json, Version=13.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed; System.Diagnostics.DiagnosticSource, Version=5.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51; System.Formats.Asn1, Version=5.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51; System.Runtime, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a; System.Threading.Tasks.Dataflow, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a`
```

Comment:
This looks a bit weird, what do you think @tamasvajk ?

Reply:
Yes, I think we would manually need to touch up the files in `_frameworks` folder, and remove any mention of `Newtonsoft.Json`.

What happens here is that `System.Diagnostics.CodeAnalysis.AllowNullAttribute` is defined in multiple assemblies, and one of them happens to be `Newtonsoft.Json`, the others are all in the runtime.

Reply:
Alright. I will just roll-back the changes made in this file as the only difference between the previous version and this one is the mentioning of Newtonsoft.Json.